src/test/java/seedu/ccacommander/storage/StorageManagerTest.java

```diff
@@ -24,7 +24,7 @@ public class StorageManagerTest {
 
     @BeforeEach
     public void setUp() {
-        JsonAddressBookStorage addressBookStorage = new JsonAddressBookStorage(getTempFilePath("ab"));
+        JsonCcaCommanderStorage addressBookStorage = new JsonCcaCommanderStorage(getTempFilePath("ab"));
```

Comment:
Should this be ccaCommanderStorage?
```suggestion
        JsonCcaCommanderStorage ccaCommanderStorage = new JsonCcaCommanderStorage(getTempFilePath("ab"));
```

Reply:
resolved